packages/endpoints-organizations-registrar/src/controllers/invitations/controller.js

```diff
@@ -74,7 +74,15 @@ const invitationsController = async (fastify) => {
             errorCode: 'invitation_expired',
           });
         }
-        return { invitation };
+        const inviterOrganization = await repos.organizations.findById(
+          invitation.inviterId
+        );
+        return {
+          invitation: {
+            ...invitation,
+            inviterDid: inviterOrganization.didDoc.id,
```

Comment:
This is just so ugly. I am thinking that the best way forward is to revert the introduction of `inviterId`. I think that Sertifier directly use the invitation API which means removing `inivterDid` needs to be handled with care

These halfway houses leave the code is a state of serious messiness.

Reply:
right, inviterId removed, see [c3c96a7](https://github.com/velocitycareerlabs/velocitycore/pull/34/commits/c3c96a7bf701683de8a1d319cf79336f0d6f309a)